examples/crna-kitchen-sink/package.json

```diff
@@ -3,15 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "devDependencies": {
-    "@storybook/addon-actions": "file:../../addons/actions",
-    "@storybook/addon-links": "file:../../addons/links",
-    "@storybook/addon-options": "file:../../addons/options",
-    "@storybook/addon-storyshots": "file:../../addons/storyshots",
-    "@storybook/addons": "file:../../lib/addons",
-    "@storybook/channels": "file:../../lib/channels",
-    "@storybook/channel-postmessage": "file:../../lib/channel-postmessage",
-    "@storybook/react-native": "file:../../app/react-native",
-    "@storybook/ui": "file:../../lib/ui",
+    "@storybook/addon-actions": "file:../../packs/storybook-addon-actions.tgz",
```

Comment:
@tmeasday This is kinda your domain... This works for me and the CI, and this seems to actually work well in `npm`. Yeah, `NPM`... I hope not to be introducing issues with this. Tested this locally on `test-cra` and actually found some issues and fixed them. Please let me know if you have any objections to this.
//cc @shilman